test/parallel/test-npm-install.js

```diff
@@ -58,6 +58,6 @@ function handleExit(error, stdout, stderr) {
   assert.strictEqual(code, 0, `npm install got error code ${code}`);
   assert.strictEqual(signalCode, null, `unexpected signal: ${signalCode}`);
   assert.doesNotThrow(function() {
-    fs.accessSync(`${installDir}/node_modules/package-name`);
+    common.fileExists(`${installDir}/node_modules/package-name`);
```

Comment:
`common.fileExists()` doesn't throw, so `assert.doesNotThrow()` is redundant. Either leave as is, or assert on the return value from `common.fileExists()`.

Side note: `common.fileExists()` isn't necessary anymore. It was added when `fs.exists()` was scheduled for deprecation and removal.

Reply:
should is maybe be replaced with fs.exists?

Do you think this PR should be closed? If so should we maybe have @df1 remove some instances of common.fileExists?

Reply:
@MylesBorins  do you mean `fs.stat()` or `fs.access()`? 
Since `fs.exists()` is deprecated.

Reply:
Sorry for saying `fs.exists()`, when I meant `fs.existsSync()`. I'd be +1 to removing `common.fileExists()` and replacing it with `fs.existsSync()`.

This particular change should either be backed out, or updated to use `existsSync()` without `assert.doesNotThrow()`.